### PR TITLE
[MAIN] [SRVKS-1123] Performance patches 

### PR DESCRIPTION
**Pull request by skonto**
**What this PR does / why we need it**:
- Adds patches to main for performance tests

### CODE REVIEW

File: openshift/perf-tests.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+set -x
+env
+
+failed=0
+
+git apply "$(dirname "$0")/performance/patches/*"
+
+make generate-dockerfiles
+
+go get github.com/elastic/go-elasticsearch/v7@v7.17.10
```
Comment (ReToCode) [inline openshift/perf-tests.sh:15]: 👍

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

/hold for CI stuff (although not relevant here). Feel free to unhold.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Comment by skonto** [comment]
Images built successfully here: https://github.com/openshift/release/pull/43710.

**Comment by skonto** [comment]
All runs pass on https://github.com/openshift/release/pull/43710.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Pull request merged**